docs/readme.md

```diff
@@ -14,16 +14,23 @@
 
 2. Bauen der *.war Files: (mvn sollte im Path liegen)
   1. Frontend:
-    - In das Verzeichnis /fronend mit der git bash wechseln
+    - In das Verzeichnis /frontend mit der git bash wechseln
     - Mit `mvn clean install` das Frontend bauen
   2. Backend:
     - In das Verzeichnis /backend mit der git bash wechseln
     - Mit `mvn clean install` das Backend bauen
+  3. Dateien backend.war und frontend.war von dem Verzeichnis js_menus\target\autodeploy in das Verzeichnis js_menus\target verschieben. Der Ordner autodeploy sollte nun leer sein.
```

Comment:
do not document workarounds if you can cure the cause.
update the buildDir parameter in the pom.xml for front- and backend such that it builds to the target directory and not the autodeploy directory